render/detailed/connections.go

```diff
@@ -117,36 +203,37 @@ func incomingConnectionsSummary(topologyID string, r report.Report, n report.Nod
 		TopologyID:  topologyID,
 		Label:       "Inbound",
 		Columns:     columnHeaders,
-		Connections: connectionRows(r, ns, counts, isInternetNode(n)),
+		Connections: connections.rows(r, ns, isInternetNode(n)),
 	}
 }
 
 func outgoingConnectionsSummary(topologyID string, r report.Report, n report.Node, ns report.Nodes) ConnectionsSummary {
 	localEndpoints := endpointChildrenOf(n)
+	connections := newConnections()
 
 	// For each node which has an edge FROM me
-	counts := map[connection]int{}
 	for _, id := range n.Adjacency {
 		node, ok := ns[id]
 		if !ok {
 			continue
 		}
 
-		remoteEndpointIDs := endpointChildIDsOf(node)
+		remoteEndpointIDs, remoteEndpointIDCopies := endpointChildIDsAndCopyMapOf(node)
 
 		for _, localEndpoint := range localEndpoints {
 			_, localAddr, _, ok := report.ParseEndpointNodeID(localEndpoint.ID)
 			if !ok {
 				continue
 			}
-
 			for _, remoteEndpointID := range localEndpoint.Adjacency.Intersection(remoteEndpointIDs) {
+				if remoteEndpointIDCopy, ok := remoteEndpointIDCopies[remoteEndpointID]; ok {
+					remoteEndpointID = remoteEndpointIDCopy
```

Comment:
I think a comment is in order here to indicate we need this to make the destination unique.